.github/workflows/project-updater.yml

```diff
@@ -0,0 +1,23 @@
+name: Update projects when an issue/PR is created/labeled
```

Comment:
We want something about "projects" being "updated", and possibly "GitHub projects" or "(beta) projects" since "projects" is a bit generic.  Mentioning "labels" would also be good, since they are the trigger of the action.

Is "Update GH projects on label change" good?  The last word might not fit, but it's not essential.  Otherwise I can just drop the "GH" and it should fit.